File: Dante1/Module1.01/Zadanie1.12/main.c
#include <stdio.h>

int main()
{
    float number;
    printf("Enter number: ");

    int check = scanf("%f",&number);
    if(check != 1)
    {
        printf("Incorrect input");
        return 1;
    }

    printf("%f\n%f",number, number * 10);
    return 0;
}

File: Dante1/Module1.01/Zadanie1.12/main.zig
const std = @import("std");

pub fn main() !void {
    const stdin = std.io.getStdIn().reader();
    const stdout = std.io.getStdOut().writer();

    try stdout.print("Enter number: ", .{});

    var buf: [100]u8 = undefined;
    const line_opt = try stdin.readUntilDelimiterOrEof(&buf, '\n');

    const line = line_opt orelse {
        try stdout.print("No input\n", .{});
        return;
    };

    const number = std.fmt.parseFloat(f32, line) catch {
        try stdout.print("Incorrect input\n", .{});
        return;
    };

    try stdout.print("{d}\n{d}\n", .{ number, number * 10 });
}
